auth/api/iam/api.go

```diff
@@ -118,30 +116,21 @@ func (r Wrapper) HandleTokenRequest(ctx context.Context, request HandleTokenRequ
 		// Options:
 		// - OpenID4VCI
 		// - OpenID4VP, vp_token is sent in Token Response
+		panic("not implemented")
```

Comment:
now we can crash any live node? There's an `unsupported_grant_type` error.

Reply:
Ah, it's a TODO, but we can return `unsupported_grant_type` indeed.

(node won't crash, the HTTP handler (goroutine) will)